source/MaterialXGraphEditor/Main.cpp

```diff
@@ -98,6 +98,7 @@ int main(int argc, char* const argv[])
     }
 
     std::string materialFilename = "resources/Materials/Examples/StandardSurface/standard_surface_marble_solid.mtlx";
+    bool isDefaultFilename = true;
```

Comment:
The initially loaded content should not be considered an explicitly "user" loaded file. So CTRL-S won't accidently overwrite it.